saleor/graphql/utils.py

```diff
@@ -14,6 +15,17 @@ def get_node(info, id, only_type=None):
     return node
 
 
+def get_nodes(ids, graphene_type):
+    """Return queryset of nodes of proper type."""
+    pks = []
+    for graphql_id in ids:
+        _type, _id = from_global_id(graphql_id)
+        assert str(graphene_type) == _type, (
+            'Must receive an {} id.').format(graphene_type._meta.name)
+        pks.append(_id)
+    return graphene_type._meta.model.objects.filter(pk__in=pks)
```

Comment:
Don't we care about the order? Do we want to return `None` where no match exists?

Reply:
I made this with mutations in mind, where we are providing few global_id, for example products, and want to add them to collection - in this case, for me the order does not matter.
I do not recall such case, when we care about the order.
I will add case for `None`, similar to `get_node` function.

Reply:
It feels natural to return an empty list if no matches are found.

Reply:
It may or may not. For example if I send a list of five Node IDs, is it natural for the API to silently only assign three of them or are there cases where a missing object is a validation error?

Reply:
@elwoodxblues Then why do we raise `Exception` in `get_node` if no node was found? I find it a bit inconsistent. For me it is more elaborate to communicate, that ids are wrong than just return empty list.

@patrys I was thinking about how to handle such case too. Do you think it would be better to remain it as it is or do another check, for example for length of input list and output queryset and raising an error if they are not equal (maybe with message, which ids were not found)?

Reply:
I don't think comparing length will work. What if I pass `ids=[1, 1, 1]`?